academy/controllers/main.py

```diff
@@ -1,16 +1,17 @@
 from odoo import http
```

Comment:
you could do this if only `request` is needed
```suggestion
from odoo.http import request
```